specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/InterfaceEndpointProfiles.json

```diff
@@ -0,0 +1,416 @@
+            "description": "Successfully deleted the interface endpoint profile."
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 OperationIdNotFound - The operation with Id does not exist.\n\n * 409 OperationCancelled - The operation has been cancelled by user.\n\n * 409 OperationInterrupted - The operation on the resource could not be completed because it was interrupted by another operation on the same resource.\n\n * 500 OperationTimedOut - The operation timed out and automatically rolled back. Please retry the operation."
+          },
+          "202": {
+            "description": "Accepted"
+          },
+          "204": {
+            "description": "The specified interface endpoint profile does not exist."
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Deletes interface endpoint profile.": {
+            "$ref": "./examples/InterfaceEndpointProfileDelete.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/interfaceEndpointProfiles": {
+      "get": {
+        "tags": [
+          "InterfaceEndpointProfiles"
+        ],
+        "description": "Gets a list of interface endpoint profiles attached to a server.",
+        "operationId": "InterfaceEndpointProfiles_ListByServer",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Succesfully retrieved the list of interface endpoint profiles.",
+            "schema": {
+              "$ref": "#/definitions/InterfaceEndpointProfilesListResult"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 ResourceNotFound - The requested resource was not found."
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "List of interface endpoint profiles on server.": {
+            "$ref": "./examples/InterfaceEndpointProfileListByServer.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "InterfaceEndpointProfileProperties": {
+      "description": "The properties of a interface endpoint profile.",
+      "required": [
+        "virtualNetworkSubnetId"
+      ],
+      "type": "object",
+      "properties": {
+        "virtualNetworkSubnetId": {
```

Comment:
Please ensure the appropriate linked access check are been performed in the manifest.

Reply:
yes we have it in manifest.